Resources/Private/Templates/Layouts/Default.html

```diff
@@ -1,3 +1,3 @@
-
 <f:render partial="Navigation/MainNavigation.html" arguments="{_all}"/>
+<f:render partial="Navigation/Breadcrumb.html" arguments="{_all}"/>
 <f:render section="Main" />
```

Comment:
Small style note: Self-closing tags are sometimes with and sometimes without space

Reply:
Done